dashboard/key-visualizer-monitoring-tool.md

```diff
@@ -1,34 +1,33 @@
 ---
-title: Key Visualizer
-summary: Learn how to monitor traffic using the Key Visualizer tool.
+title: Key Visualizer Page
+summary: Learn how to use Key Visualizer to monitor traffic.
 category: how-to
+aliases: ['/docs/dev/key-visualizer-monitoring-tool/']
 ---
 
-# Key Visualizer
+# Key Visualizer Page
 
-Key Visualizer is a feature for analyzing the usage of TiDB and troubleshooting the traffic hotspot. The feature generates a visual report of the metrics of the TiDB cluster over a period of time. With this visual report, you can quickly and intuitively observe the overall cluster hotspot and traffic distribution.
+The Key Visualizer page of TiDB Dashboard is used to analyze the usage of TiDB and troubleshoot traffic hotspots. This page visually shows the traffic of the TiDB cluster over a period of time.
 
-## Access Key Visualizer
+## Access Key Visualizer page
 
-As one of the offerings of TiDB Dashboard, Key Visualizer is directly integrated into PD instances and does not require a separate deployment. You can access TiDB Dashboard on any PD instance via your browser by using the following address:
+You can use one of the following two methods to access the Key Visualizer page:
 
-{{< copyable "" >}}
+* After logging into TiDB Dashboard, click **Key Visualizer** on the left navigation menu:
 
-```
-http://PDAddress:PDPort/dashboard
-```
+![Access Key Visualizer](/media/dashboard/dashboard-keyviz-access.png)
+
+* Visit <http://127.0.0.1:2379/dashboard/#/keyviz> in your browser. Replace `127.0.0.1:2379` with the actual PD instance address and port.
 
 > **Note:**
 >
-> + If multiple PD instances exist, you can use any PD instance address to access TiDB Dashboard.
-> + By default, `PDPort` is `2379`. If you have modified the corresponding PD parameter during deployment, use the modified PD port.
-> + You need to use the `root` account of TiDB to log into TiDB Dashboard.
+> The Key Visualizer feature is disabled by default. If you use this feature for the first time, You need to manually enable it. To enable this feature, follow the interface guide or see the [Settings](#settings) section of this document.
```

Comment:
This feature is currently enabled by default, please delete this part.

Reply:
Deleted.